actionpack/lib/action_dispatch/middleware/static.rb

```diff
@@ -13,10 +16,10 @@ module ActionDispatch
   # located at `public/assets/application.js` if the file exists. If the file
   # does not exist a 404 "File not Found" response will be returned.
   class FileHandler
-    def initialize(root, cache_control)
+    def initialize(root, cache_control, access_control_headers)
```

Comment:
Although this implementation works, I'm not sure about this interface since `Cache-Control` is also part of the HTTP headers and the third argument seems redundant. I guess this interface could be just`default_headers`.